chat-plugins/uno.js

```diff
@@ -596,6 +597,23 @@ exports.commands = {
 			this.modlog('UNO TIMER', null, `${amount} seconds`);
 		},
 
+		autostart: function (target, room, user) {
+			if (!this.can('minigame', null, room)) return;
+			if (!room.game || room.game.gameid !== 'uno') return this.errorReply("There is no UNO game going on in this room right now.");
+			let amount = parseInt(target);
+			if (!amount || amount < 30 || amount > 600) return this.errorReply("The amount must be a number between 30 and 600.");
+			room.game.autostartTimer = amount;
+			if (room.game.autostartTimer) {
+				clearTimeout(room.game.autostartTimer);
+			}
+			room.game.autostartTimer = setTimeout(() => {
```

Comment:
imo it's cleaner just to clear it in destroy() and onstart()

Reply:
oh